pom.xml

```diff
@@ -81,33 +81,23 @@
             <artifactId>cds-services-api</artifactId>
         </dependency>
 
-        <!-- JUnit 5 -->
         <dependency>
             <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <version>5.10.2</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
Why is this removed? Are you able to run UTs without this?

Reply:
Added back